sources/platform/runtime/machine-image/cli-versions.md

```diff
@@ -25,9 +25,12 @@ This page describes the versions available for each CLI and the Machine Image(s)
 
 ## Ansible
 
-| Version  |  Machine Images  
-|----------|---------
-| 2.3.0.0  | [v5.6.1](/platform/runtime/machine-image/ami-v561/) and later
+### 
+
+| Version  |  OS  | Machine Images | 
+|----------|--------|--------|
+| 2.4.4 | - Ubuntu 14.04<br> - Ubuntu 16.04 <br> - Centos 7 <br> - Ubuntu 16.04 (aarch32)<br> - Ubuntu 16.04 (aarch64)| [v6.4.4](/platform/runtime/machine-image/ami-v644/)
```

Comment:
Could we change Centos to CentOS. Also, we might want to explicitly mention x86_64.